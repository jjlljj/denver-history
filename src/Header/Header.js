import React, { Component } from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import './Header.css';

export class Header extends Component {
  constructor(props){
    super(props);
    this.state = {

    };
  }

  render() {
    return (
      <div>
        i am Header
      </div>
    )
  }
}

export default connect(null, null)(Header)
