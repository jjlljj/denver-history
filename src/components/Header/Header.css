.site-header {
  align-items: center;
  background: green;
  display: flex;
  height: 60px;
  justify-content: center;
}

.site-header {
  font-size: 30px;
}
