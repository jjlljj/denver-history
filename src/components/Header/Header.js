import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Header.css';

export default class Header extends Component {
  constructor(props){
    super(props);
    this.state = {

    };
  }

  render() {
    return (
      <header className="site-header">
        <h1>Denver History</h1>
      </header>
    )
  }
}
