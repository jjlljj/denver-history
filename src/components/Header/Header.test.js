import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {

  it('renders without crashing', () => {
    const renderedComponent = shallow(<Header />)
    expect(renderedComponent).toBeDefined()
  });
})
