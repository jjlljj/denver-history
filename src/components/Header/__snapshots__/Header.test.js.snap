// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        replace={false}
        to="/"
>
        <h1>
                Historic Denver
        </h1>
</Link>,
      "className": "site-header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <h1>
          Historic Denver
</h1>,
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Historic Denver",
        },
        "ref": null,
        "rendered": "Historic Denver",
        "type": "h1",
      },
      "type": [Function],
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          replace={false}
          to="/"
>
          <h1>
                    Historic Denver
          </h1>
</Link>,
        "className": "site-header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <h1>
            Historic Denver
</h1>,
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Historic Denver",
          },
          "ref": null,
          "rendered": "Historic Denver",
          "type": "h1",
        },
        "type": [Function],
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
